features/iterator-methods.yml

```diff
@@ -0,0 +1,18 @@
+name: Iterator methods
+description: The constructor and the `drop()`, `every()`, `filter()`, `find()`, `flatMap()`, `forEach()`, `from()`, `map()`, `reduce()`, `some()`, `take()` and `toArray()` methods on the `Iterator` object.
+spec: https://tc39.es/proposal-iterator-helpers/
+group: iterators
+compat_features:
+  - javascript.builtins.Iterator.Iterator
```

Comment:
Just for posterity: I wondered by `javascript.builtins.Iterator` wasn't included here. Ultimately, I found that it makes sense: BCD tells a half-truth in that all browsers support `Iterator` in the sense that they implement iterators. But don't actually expose `Iterator` as a defined object, at least not until they ship `Iterator()`. The BCD and MDN docs are extremely confusing on this point.